cli/cli_app.go

```diff
@@ -715,16 +717,27 @@ func processHooks(hooks []config.Hook, terragruntOptions *options.TerragruntOpti
 
 			actionToExecute := curHook.Execute[0]
 			actionParams := curHook.Execute[1:]
-			_, possibleError := shell.RunShellCommandWithOutput(
-				terragruntOptions,
-				workingDir,
-				false,
-				false,
-				actionToExecute, actionParams...,
-			)
-			if possibleError != nil {
-				terragruntOptions.Logger.Errorf("Error running hook %s with message: %s", curHook.Name, possibleError.Error())
-				errorsOccured = multierror.Append(errorsOccured, possibleError)
+
+			if actionToExecute == "tflint" {
+				actionParams := curHook.Execute[0:]
+
+				err := tflint.RunTflintWithOpts(terragruntOptions, terragruntConfig, actionParams)
+				if err != nil {
+					terragruntOptions.Logger.Errorf("Error running hook %s with message: %s", curHook.Name, err.Error())
+					multierror.Append(errorsOccured, err)
```

Comment:
This should be `errorsOccurred = multierror.Append(...)`. See the analogous code in the else statement.

Reply:
Good catch! Thanks